data/pigui/libs/face.lua

```diff
@@ -36,7 +36,7 @@ local function rerollFaceDesc(oldDesc)
 		FEATURE_NOSE = Engine.rand:Integer() % 2^31,
 		FEATURE_MOUTH = Engine.rand:Integer() % 2^31,
 		FEATURE_HAIRSTYLE = Engine.rand:Integer() % 2^31,
-		FEATURE_ACCESSORIES = Engine.rand:Integer() % 2^31,
+		FEATURE_ACCESSORIES = Engine.rand:Integer() % 4 == 0 and Engine.rand:Integer() % 2^31 or 0, -- accessory on one out of four faces
```

Comment:
I guess the `and` branch can also generate an empty accessory?

Reply:
Correct. With the current number of accessories every eight png will be invisible.